### ISSUE TITLE
Multiple Paragraphs Worth of Help Text Inside of H3

### ISSUE THREAD

**Issue by mtribone** [psu-libraries/scholarsphere-3#1062]
When editing a fileset and selecting the "Permissions" tab, there are multiple paragraphs worth of help text in various sizes and styles inside of the "Sharing With" header. This is not good HTML structure or helpful to users of assistive technologies. And contains a clickable button.

![screen shot 2017-10-26 at 12 34 40 pm](https://user-images.githubusercontent.com/4163828/32070541-b72853ce-ba5a-11e7-8557-2ac7eeb19fc8.png)

**Comment by carolyncole** [comment]
They should be `<p>` tag and remove clickable button.  Use the collaborators tab as an example
![Screen Shot 2017-12-13 at 10.37.02 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/54e5f67e04dc270b004037e9/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c57e0b76216455f06b7191a379ef0c2856415f4cbc30254d58eeb9cb74fce36f34985619e8d5380b22c76815febe811fbb09bb298b4f83b1ba8a1ac7fa2137a5dad11b61435c52dcbd4f3fb29842c259f34ae24c52ad173e6aee7e3f6e0a.png)

![Screen Shot 2017-12-13 at 10.37.24 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/54e5f67e04dc270b004037e9/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c57e0b76216455f06b7191a379ef0c2856415d4abc30254d58eeb9cb74fce36f34985619e8d5380b22c76815febe811fbb09bb298b4f83b1ba8a1ac7fa2137a5dad11b61435c52dcbd4f3fb29842c259f34ae24c52ad173f66e574326e0a.png)
